packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -263,6 +270,11 @@ class BottomSheetRangeCell extends Component {
 							value={ inputValue }
 						/>
 					) }
+					{ shouldDisplayTextInput && inputSuffix && (
```

Comment:
Recently a new component, `UnitControl` was [introduced](https://github.com/WordPress/gutenberg/pull/26161). Maybe we can use it instead of adding this?

Reply:
@geriux I'll check it out. Thanks!

Reply:
@geriux after reviewing `UnitControl`, I think we'd need to (1) request Thomas' input on whether changing from a slider to a stepper is desired and (2) update `UnitControl` to support displaying a single unit and avoid the user tapping the unit to display a unit selection of one option (i.e. there will never be a need to change the unit from percentage to anything else).

Do you think it's worthwhile to explore this?

Reply:
I see, thanks for looking into it! I think we could just leave this implementation for now =)

Reply:
@geriux I overlooked and misunderstood that `UnitControl` already supports rendering `RangeCell`. So, this should be easy to accomplish. I plan to implement it. Thanks.